docs/content/preview/reference/configuration/yb-master.md

```diff
@@ -425,59 +425,47 @@ Default: `3`
 
 The number of shards per YB-TServer for each YCQL table when a user table is created.
 
-Default: `-1` (the value is calculated at runtime). For servers with up to two CPU cores, the default value is considered as `4`. For three or more CPU cores, the default value is considered as `8`. Local cluster installations, created with `yb-ctl` and `yb-docker-ctl`, use a value of `2` for this flag. Clusters created with `yugabyted` use a default value of `1`.
-
-{{< note title="Important" >}}
-
-This value must match on all `yb-master` and `yb-tserver` configurations of a YugabyteDB cluster.
-
-{{< /note >}}
-
-{{< note title="Important" >}}
-
-When the value is set to *Default* (`-1`), then the server internally *updates* the flag with intended value during startup prior to version `2.18` and the flag remains *unchanged* starting from version `2.18`.
-
-{{< /note >}}
+Default: `-1` (the value is calculated at runtime).
+\
+For servers with up to two CPU cores, the default value is considered as `4`.
+\
+For three or more CPU cores, the default value is considered as `8`.
+\
+Local cluster installations created with `yb-ctl` and `yb-docker-ctl` use a value of `2` for this flag.
+\
+Clusters created with `yugabyted` use a default value of `1`.
 
 {{< note title="Note" >}}
 
-On a per-table basis, the [`CREATE TABLE ... WITH TABLETS = <num>`](../../../api/ycql/ddl_create_table/#create-a-table-specifying-the-number-of-tablets) clause can be used to override the `yb_num_shards_per_tserver` value.
-
-{{< /note >}}
-
-{{< note title="Note" >}}
-
-If [`enable_automatic_tablet_splitting`](#enable-automatic-tablet-splitting) is `true`: the default value is considered as `1` and tables will begin with 1 tablet *per node*; from version `2.18` for servers with up to 4 CPU cores, the value *is not defined* and tables will begin with 1 tablet (for servers with up to 2 CPU cores) or 2 tablets (for servers with up to 4 CPU cores) *per cluster*.
+- This value must match on all `yb-master` and `yb-tserver` configurations of a YugabyteDB cluster.
+- If the shard value is set to *Default* (`-1`), then the server internally *updates* the flag with intended value during startup prior to version `2.18` and the flag remains *unchanged* starting from version `2.18`.
```

Comment:
These are actually "Important" notes, and not just notes. Also having 4 different note blocks is looking fine to me 
<img width="867" alt="image" src="https://github.com/yugabyte/yugabyte-db/assets/79331145/7844993d-2740-4aed-9ac3-ce1f6d670978">

cc @rthallamko3 to comment

Reply:
Suggested update:
<img width="864" alt="image" src="https://github.com/yugabyte/yugabyte-db/assets/79331145/065fb465-01fc-4509-a9f3-bf45b3056a0a">